.travis.yml

```diff
@@ -74,7 +74,7 @@ jobs:
 
   # Clippy linting
   - env: FEATURE=clippy
-    rust: nightly-2018-08-30
+    rust: 1.29.1 # Use fresh clippy
```

Comment:
I'm insisting on adding additional Travis job to check compatibility with 1.27.2. [as in core](https://github.com/exonum/exonum/blob/master/.travis.yml#L174). The reason is, Java Bindings depends on anchoring, and therefore anchoring should support 1.27.2 until next Rust release. This PR successfully builds with 1.27.2

cc @dmitry-timofeev @viktorbezhenar @DarkEld3r